python/setup.py

```diff
@@ -121,11 +121,15 @@
 if "--singlegpu" in sys.argv:
     exc_list.append('cuml/linear_model/linear_regression_mg.pyx')
     exc_list.append('cuml/decomposition/tsvd_mg.pyx')
+    exc_list.append('cuml/neighbors/nearest_neighbors_mg.pyx')
     exc_list.append('cuml/cluster/kmeans_mg.pyx')
     sys.argv.remove('--singlegpu')
 else:
     libs.append('cumlprims')
 
+    sys_include = os.path.dirname(sysconfig.get_path("include"))
+    include_dirs.append("%s/cumlprims" % sys_include)
```

Comment:
this should not be needed with line https://github.com/rapidsai/cuml/blob/f707319a05a31b097f63340f4c26e504dc8e17ed/python/setup.py#L111
particularly since the includes are already in the form: 
```
cdef extern from "cumlprims/opg/matrix/part_descriptor.hpp"
```